src/templates/rust-all/.gitignore.tftpl

```diff
@@ -5,10 +5,20 @@
 # will have compiled files and executables
 debug/
 target/
+
+*.env
+
 # Local cargo cache generated by make targets
 .cargo/registry
 
-# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
+%{ if type == "svc" }
+server/.cargo/registry
+client/.cargo/registry
+client/Cargo.lock
```

Comment:
Will there only be a Cargo.lock for the client?

Reply:
I guess Cargo.lock is only generated at the top level with this approach, so we don't need to ignore client/Cargo.lock

The toplevel Cargo.lock should continue to be tracked in the GitHub because of the server binary

Will change

Reply:
https://github.com/Arrow-air/tf-github/pull/34/commits/8f710a84089f9d4cd7907b447d47d911e0caf123